torch/_dynamo/variables/builtin.py

```diff
@@ -306,6 +332,63 @@ def call_function(
             )
             return out
 
+        # Handle functions that are reversible (e.g. __add__ / __radd__)
+        reversible_binops = self._reversible_binops()
+        if self.fn in reversible_binops:
+            assert len(kwargs) == 0 and len(args) == 2
+
+            # Call reverse for user-defined variables, if any
+            forward_name, reverse_name = reversible_binops[self.fn]
+            if any([isinstance(arg, UserDefinedVariable) for arg in args]):
+                if isinstance(args[0], UserDefinedVariable):
+                    return args[0].call_method(tx, forward_name, args[1:], kwargs)
+                else:
+                    return args[1].call_method(tx, reverse_name, [args[0]], kwargs)
+
+            # Insert operator directly into the graph for dynamic shape args
+            if self._dynamic_args(*args, **kwargs):
+                return self._dyn_proxy(tx, *args, **kwargs)
+
+            # Handle special cases for list / tuple.
+            # TODO: Develop a proper abstraction to avoid this nonsense
```

Comment:
This is the part I'm talking about - specific customizations for lists / tuples that can no longer be handled through the `get_method()` abstraction